tests/integration/tests/test_external_certificates.py

```diff
@@ -166,6 +175,11 @@ def check_nginx_pod_runs(instance: harness.Instance):
     )
 
 
+def delete_nginx_pod(instance: harness.Instance):
+    manifest = MANIFESTS_DIR / "nginx-pod.yaml"
+    instance.exec(["k8s", "kubectl", "delete", "-f", "-"], input=manifest.read_bytes())
```

Comment:
nit: I think we can write this like:
```suggestion
    manifest = MANIFESTS_DIR / "nginx-pod.yaml"
    instance.exec(["k8s", "kubectl", "delete", "-f", string(manifest)])
```

Reply:
Sure, I'll change this to `str(manifest)`